### DISCUSSION TITLE
Option to Disable Manga Related Features and Other Ideas

### DISCUSSION THREAD

**Discussion by GusNa** [aniyomiorg/aniyomi#194] [Ideas]
Honestly, I'm more interested in all the anime features of Aniyomi than any of the Manga features. For manga, I still use Tachiyomi.

Because Tachiyomi is a bigger, more popular project with more contributors and people reviewing the code, it will in all likelihood have more frequent updates, better security, and better manga related features going forward than a smaller project like Aniyomi can consistently deliver. No offense is meant by this, I'm still loving this app very much for the anime features.

With this in mind, it might be nice to have a toggle button in the "More" section to disable the manga related features for people who only want anime features.
<details>
  <summary>Expand this section to see some changes that could happen when the button is toggled.</summary>
  
  - Hide the `Manga` library button.
  - Hide `Updates > Manga` tab.
  - Hide the `Browse > Manga Sources` and `Browse > Manga Extensions` tabs.
  - Hide the `More > History > Manga` tab.
  - Hide the `More > Download queue > Manga` tab.
  - Hide the `More > Manga Categories` menu item.
  - Hide the `More > Settings > Library > Categories > Edit manga categories and Default manga category` buttons.
  - Hide the `More > Settings > Library > Global update > Manga Categories` button.
  - Hide the `More > Settings > Reader` menu item.
  - Hide the `More > Settings > Downloads > Delete chapters/episodes > Excluded manga categories` button.
  - I probably missed some stuff like cleanup and wording changes but I think I got most of the big features.
</details>

---

That being said, it might be a good decision to narrow development focus to only anime and remove all of the manga features from Aniyomi, much like [LNReader](https://github.com/LNReader/lnreader) does for Light Novels. This might save a lot of time simply by not needing to maintain manga related features going forward. Another option would be to try and contribute the anime related features directly to Tachiyomi instead of keeping the projects separate from each other.

Either of those two development paths would probably be large undertakings and would need a lot of discussion from the developers and the community before any decisions were made. I'm pretty inexperienced in actually using GitHub in general and I don't really know how much work either would entail or what rules would need to be followed, so forgive me if my ideas seem silly/rude from you developer's perspectives. I just thought I'd get my ideas out there for everyone to discuss!

**Comment by Roswaal12** [comment]
You can use Animiru for that.

https://github.com/Quickdesh/Animiru
